source/partials/_video-cover.html.slim

```diff
@@ -9,9 +9,8 @@
 
 ruby:
   if item.video.present?
-    nocookie_url = item.video.url.gsub("youtube.com/watch?v=",
-                   "youtube-nocookie.com/embed/")
-    minheight = item.video.height < 400 ? 400 : video.height
+    nocookie_url = item.video.url.gsub("youtube.com/watch?v=","https://youtu.be/")
```

Comment:
We should keep the nocookie domain here

Reply:
We implemented the consent on youtube 3rd party cookies, so we can get the rid of old  (not working) no-cookie url

Reply:
@Lorezz using youtube-nocookie.com is still slightly better, we had internal discussions about that one and came out with a best practice over the years, take a look at (unfortunately not accessible from the outside):

* https://italiandigitalteam.slack.com/archives/C3JBPM1GF/p1664355801800239?thread_ts=1664277841.817499&cid=C3JBPM1GF
* https://italiandigitalteam.slack.com/archives/CDY0TDV9P/p1645724804219149?thread_ts=1643799449.785159&cid=CDY0TDV9P

from starters, but there's most like more discussions if you look around.

Reply:
ok, I'll switch back to nocookie url